src/accessibility/WCAG-2.2/index.md

```diff
@@ -0,0 +1,393 @@
+---
+title: Changes to the Design System to meet WCAG 2.2
+description: Outlines the current principles and work needed to improve the accessibility of the GOV.UK Design System
+section: Accessibility
+layout: layout-pane.njk
+showPageNav: true
+order: 2
+---
+
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+W3C owns and manages the [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/).
+
+In 2023, the Design System team assessed and updated the GOV.UK Design System to comply with the new criteria and recommendations given in WCAG 2.2.
+
```

Comment:
Should there also be a sentence on the fact that code changes were made as par of GOV.UK Frontend v5?
Something to help clarify that it's not just an assessment we've done, but that getting some of the WCAG 2.2 DS changes will require teams to move to v5.

Reply:
Good shout! Also a chance to link to the v5 page in Frontend docs.

Reply:
Is this that v5 page in Frontend docs? https://frontend.design-system.service.gov.uk/changes-to-govuk-frontend-v5/

If so, I also just submitted a PR to add a section on that page for the accessibility-specific changes:
https://github.com/alphagov/govuk-frontend-docs/pull/415